### DISCUSSION TITLE
Zen Sidebar

### DISCUSSION THREAD

**Discussion by Jpeq** [Tanay-Kar/Phi#2] [General]
Hello, I've seen you make a Load Bar for Zen mods, and I want to ask something. I'm not too good at customizing, but I absolutely love the look of your sidebar. Could you help me how could I get that as well? I really appreciate it if you would like to help me.

**Comment by Tanay-Kar** [comment]
No problem mate ,

I was thinking we should continue this on a personal forum ... Let me know where you'd like to talk 


Regards
